dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Voltage/StaticVarCompensator/SVarCControl.mo

```diff
@@ -0,0 +1,106 @@
+
+  extends Parameters.Params_Regulation;
+  extends Parameters.Params_Limitations;
+  extends Parameters.Params_CalculBG;
+  extends Parameters.Params_ModeHandling;
+
+  Modelica.Blocks.Interfaces.RealInput URef(start = URef0) "Voltage reference for the regulation in kV" annotation(
+    Placement(visible = true, transformation(origin = {-168, 26}, extent = {{-20, -20}, {20, 20}}, rotation = 0), iconTransformation(origin = {-115, 85}, extent = {{-15, -15}, {15, 15}}, rotation = 0)));
+  Modelica.Blocks.Interfaces.RealInput UPu(start = U0Pu) "Voltage at the static var compensator terminal in p.u (base UNom)" annotation(
+    Placement(visible = true, transformation(origin = {-168, -2}, extent = {{-20, -20}, {20, 20}}, rotation = 0), iconTransformation(origin = {-115, -41}, extent = {{-15, -15}, {15, 15}}, rotation = 0)));
+  Modelica.Blocks.Interfaces.RealInput QInjPu(start = -Q0Pu) "Reactive power injected by the static var compensator in p.u (base SnRef) (generator convention)" annotation(
+    Placement(visible = true, transformation(origin = {-168, -68}, extent = {{-20, -20}, {20, 20}}, rotation = 0), iconTransformation(origin = {-115, -83}, extent = {{-15, -15}, {15, 15}}, rotation = 0)));
+
+  Modelica.Blocks.Interfaces.BooleanInput selectModeAuto(start = true) "Mode selected when in manual configuration" annotation(
+    Placement(visible = true, transformation(origin = {-168, 54}, extent = {{-20, -20}, {20, 20}}, rotation = 0), iconTransformation(origin = {-115, 49}, extent = {{-15, -15}, {15, 15}}, rotation = 0)));
+  Modelica.Blocks.Interfaces.IntegerInput setMode(start = 2) "Wheter the static var compensator is in automatic configuration" annotation(
+    Placement(visible = true, transformation(origin = {-168, 84}, extent = {{-20, -20}, {20, 20}}, rotation = 0), iconTransformation(origin = {-115, 9}, extent = {{-15, -15}, {15, 15}}, rotation = 0)));
+
+  Modelica.Blocks.Interfaces.RealOutput BPu(start = B0Pu) "Susceptance of the static var compensator in p.u (base SNom)" annotation(
+    Placement(visible = true, transformation(origin = {146, -8}, extent = {{-10, -10}, {10, 10}}, rotation = 0), iconTransformation(origin = {110, 50}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Interfaces.RealOutput GPu(start = G0Pu) "Conductance of the static var compensator in p.u (base SNom)" annotation(
+    Placement(visible = true, transformation(origin = {146, -28}, extent = {{-10, -10}, {10, 10}}, rotation = 0), iconTransformation(origin = {110, -50}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+
+  Modelica.Blocks.Sources.Constant GPu_cst(k = G0Pu)  annotation(
+    Placement(visible = true, transformation(origin = {32, -42}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Regulation regulation(BMaxPu = BMaxPu, BMinPu = BMinPu,BVar0Pu = BVar0Pu, IMaxPu = IMaxPu, IMinPu = IMinPu, KCurrentLimiter = KCurrentLimiter, KG = KG, Kp = Kp, Lambda = Lambda, SNom = SNom, Ti = Ti)  annotation(
```

Comment:
extra space missing BMinPu, BVar0Pu to increase readability

Reply:
fixed